USAGE.rst

```diff
@@ -0,0 +1,137 @@
+kafkacat
+========
+
+kafkacat is a command line tool for working with Kafka. Described as "netcat for Kafka", it is a swiss-army knife of tools for inspecting and creating data in Kafka.
+
+If you already know ``kafka-console-producer`` and ``kafka-console-consumer`` then it's like those, on steroids.
+
+kafkacat is an open-source tool, available at https://github.com/edenhill/kafkacat. It is not included in Confluent Platform.
+
+``kafkacat`` the Consumer
+-------------------------
+
+Let's start simple. Give ``kafkacat`` a broker and a topic, and you'll see its contents:
+
+.. code:: shell
+
+    $ kafkacat -b localhost:9092 -t mysql_users
+    % Auto-selecting Consumer mode (use -P or -C to override)
+    {"uid":1,"name":"Cliff","locale":"en_US","address_city":"St Louis","elite":"P"}
+    {"uid":2,"name":"Nick","locale":"en_US","address_city":"Palo Alto","elite":"G"}
+    [...]
+
+
+Note: ``kafkacat`` doesn't currently support Avro.
+
+``kafkacat`` defaults to being a consumer, but you can also make this explicit with the ``-C`` flag. You can also specify how many messages you want to see with the lowercase ``-c`` and a number. For example:
+
+.. code:: shell
+
+    $ kafkacat -b localhost:9092 -t mysql_users -C -c1
+    {"uid":1,"name":"Cliff","locale":"en_US","address_city":"St Louis","elite":"P"}
+
+Seeing the value of the message is useful, but Kafka messages also have a key, which ``kafkacat`` can display with the ``-K`` argument also supplying the delimiter:
+
+.. code:: shell
+
+    $ kafkacat -b localhost:9092 -t mysql_users -C -c1 -K\t
+    1   {"uid":1,"name":"Cliff","locale":"en_US","address_city":"St Louis","elite":"P"}
+
+With ``kafkacat`` we can really dig into a topic's messages. The ``-f`` flag takes arguments specifying both the format of the output, as well as the fields to include. Here's a simple example of pretty-printing the key/value pairs for each message:
```

Comment:
suggest removing this line: `With ``kafkacat`` we can really dig into a topic's messages.`